book/src/concepts/advanced_replication/visual_interpolation.md

```diff
@@ -16,27 +16,17 @@ To solve this, `lightyear` provides the `VisualInterpolation` plugin.
 The plugin will take care of interpolating the position of the entity between the last two `FixedUpdate` ticks, thus making sure that 
 the entity is making smooth progress on every frame.
 
-There are 3 main ways we can interpolate the component:
```

Comment:
I'd prefer to keep these 3 methods in the write-up; but yes maybe we can make it clearer that lightyear uses the first one